CHANGELOG.md

```diff
@@ -0,0 +1,246 @@
+# Change Log for AWS IoT Embedded C SDK
+
+## 202009.00 (September 14, 2020)
+
+- This release introduces re-designed MQTT client library, JSON parser library and the AWS IoT Shadow client library.
+- The AWS IoT Jobs service client library is not part of this release. It will be part of a future release.
```

Comment:
I dont think we need to call this out aspect of Jobs lib.

Reply:
Okay, have removed it. Or is there another aspect of the Jobs library that is worth mentioning?